src/utilities/browser.ts

```diff
@@ -0,0 +1,18 @@
+export function isChrome() {
```

Comment:
Seems like this file wasn't ultimately necessary

Reply:
True, but I decided to leave it in for next time. Webpack won't include it if it's not used, so it doesn't hurt anything and it might save a few minutes the next time the need arises, even if just for debugging purposes.